examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>convert I-language to you-language. "You can start with implementing..." for example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I made a few comments throughout the notebook, such as _NBI is good for prototyping, local jobs are good for debugging_, etc. The reason I intentionally used the "I" language (not even we) is because I wanted to slightly de-emphasize these recommendations. Using "we" or "you", I would be speaking for Braket to provide best practices officially. Due to some limitations in the customer experience right now(NBI editor support, local Job error log, etc), I wanted to stay away from there if possible.

Yet I do understand due to my position, whatever I say will still be more or less official recommendations.  I am good to change them to "you"s if we all agree to do so.